e2e/utils/tenv.ts

```diff
@@ -158,7 +164,31 @@ export interface Tenv {
   container(opts: ContainerOptions): Promise<Container>;
 }
 
+export async function checkDockerRunning(): Promise<void> {
+  try {
+    await docker.ping();
+  } catch (err) {
+    throw new Error('Docker is not running. Please start Docker and try again.');
+  }
+}
+
+export async function checkLocalDockerImage(image: string): Promise<boolean> {
+  try {
+    await docker.getImage(image).inspect();
+    console.log(`Local Docker image ${image} found:`);
+    return true;
+  } catch (err) {
+    console.error(`Error checking local Docker image ${image}:`, err);
+    return false;
+  }
+}
+
 export function createTenv(cwd: string): Tenv {
+  checkDockerRunning().catch(err => {
+    console.error(err.message);
+    process.exit(1);
+  });
+
```

Comment:
This is unnecessary because the test will fail anyways reporting a docker error from dockerode.

```suggestion
```